en/test_and_ci/continous_integration.md

```diff
@@ -1,3 +1,187 @@
+  - Parameter metadata
+  - Failsafe documentation
+  - uORB Message file documentation
+  - uORB Graph file
+- Deployment
+  - Update of PX4 Documentation
+  - Update on QGroundControl
+  - Update on PX4 ROS Messages
+  - Update of metadata in S3 (redundant to `deploy_all.yml` Github Action)
+
+
+## PX4 Documentation
+
+Jenkins uses the [PX4BuildBot](https://github.com/PX4BuildBot) account to upload the following metadata to the [PX4-user_guide](https://github.com/PX4/PX4-user_guide) repo when the PX4-Autopilot `main` branch is altered.
+
+- `airframes_reference.md`: Markdown for [Airframes Reference Documentation](../airframes/airframe_reference.md)
+- `parameter_reference.md`: Markdown for [Parameter Reference Documentation](../advanced_config/parameter_reference.md)
+- `modules/*.md`: Markdown for [Modules documentation](../modules/README.md)
+- `graph_*.json`: JSON defining uORB Graph, visualized in [uORB Graph](../middleware/uorb_graph.md) page
+- `msg_docs/*.md`: Markdown for [uORB Messages documentation](../msg_docs/README.md)
+- `failsafe_sim/*`: Files for [Failsafe Simulator Interface](../config/safety_simulation.md)
+
+An example commit can be found [here](https://github.com/PX4/PX4-user_guide/commit/a4ffaf5fc32df8650da614aa3129b3b90794584f).
+
+The PX4 user guide is built when there are any changes to the [PX4-user_guide](https://github.com/PX4/PX4-user_guide) repo `main` or `v1.*` branches, including the addition of the metadata above.
+This ensures ensures that the "main" version of the documentation is always up to date.
+
+Note that the documentation build is also run using Jenkins CI, and is based on the definition in [PX4-user_guide/Jenkinsfile](https://github.com/PX4/PX4-user_guide/blob/main/Jenkinsfile).
+
+
+## QGroundControl
+
+The PX4 firmware uploaded by Jenkins is used by QGroundControl for [PX4 Firmware updates](../config/firmware.md) (stable: `release/*` / beta: `main`).
+
+The Jenkins server uses the [PX4BuildBot](https://github.com/PX4BuildBot) account to upload the following metadata to QGC whenever there is an update to main or a release branch:
+
+- `airframes.xml`: Airframes metadata describing all supported airframes
+- `parameters.xml`: Parameters metadata describing all supported parameters
+
+QGC uses the above information for populating the [airframe selection](../config/airframe.md) page for PX4 and the _default_ parameter metadata information (QGC will use parameter metadata from the vehicle instead if it is available).
+An example commit of parameter metadata can be found [here](https://github.com/mavlink/qgroundcontrol/commit/7f4f3b6253fe80a881e9a91a1f5b6d960ad11834).
+
+Note that QGroundControl normally gets metadata for [Events](../concept/events_interface.md#implementation)and [Parameters](../advanced/parameters_and_configurations.md#publishing-parameter-metadata-to-a-gcs) from the vehicle (it is built into the firmware, or a URL is provided in the firmware from which it can be downloaded).
```

Comment:
Not sure if it is true to say that 'URL is provided in the firmware from which metadata can be downloaded'? @bkueng